### DISCUSSION TITLE
Tailwind Positioning

### DISCUSSION THREAD

**Discussion by m40ma0** [tailwindlabs/tailwindcss#9783] [Help]
Why does left-1/2, top-1/2, etc... not display the object in the middle of a page (eg top-1/2 displays the object exactly halfway down the page)

**Comment by akbng** [comment]
These properties won't work if the element is not positioned relative, absolute, or fixed. And also if the parent element is set to relative then the top, left will be inside that element only.
Please try: 
```html
<div class="absolute top-1/2 left-1/2 ..."></div>
```
If you want the element to be in the middle of the page all the time, replace 'absolute' with 'fixed'
